spec.html

```diff
@@ -24,20 +24,17 @@ <h1>Binary Logical Operators</h1>
   <h2>Syntax</h2>
   <emu-grammar type="definition">
     <ins>
-    CoalesceExpression[In, Yield, Await, Coalesced] :
-      LogicalORExpression[?In, ?Yield, ?Await, ?Coalesced]
-      CoalesceExpression[?In, ?Yield, ?Await, +Coalesced] `??` LogicalORExpression[?In, ?Yield, ?Await, +Coalesced]
-    </ins>
+    LogicalExpression[In, Yield, Await] :
```

Comment:
I think we need a change to ternary's grammar, too.

Reply:
That's lower down (https://github.com/tc39/proposal-nullish-coalescing/pull/45/files/a4f09e6d84b9fda817b1e8ad77e4d33069515652#diff-3540caefa502006d8a33cb1385720803R96), but the diff doesn't show enough context. Is that what you meant?

Reply:
Missed that, I only saw the `IsFunctionDefinition` and below.